### PR TITLE
Minor edit to loss function

### PR DESCRIPTION
**Pull request by vratins**
We train the shipped checkpoints with uniform loss, and no weighting at any time points -- hence removing redundant code. 

## Summary by CodeRabbit

* **Bug Fixes**
  * Updated velocity-error calculations to use consistent, unweighted mean squared error during training and validation.
  * Improved per-graph diagnostic loss reporting by averaging error across atoms.
  * Retained safeguards for detecting invalid training values such as NaNs.

### CODE REVIEW

**Review by DorisMai — APPROVED** [review]
lgtm. just double checking that other hyperparameters are not stale, like lr and grad clipping
